stable/jenkins/Chart.yaml

```diff
@@ -2,9 +2,9 @@
 apiVersion: v1
 name: jenkins
 home: https://jenkins.io/
-version: 2.5.3
+version: 2.5.4
 appVersion: lts
-description: Open source continuous integration server. It supports multiple SCM tools
+description: DEPRECATED - Open source continuous integration server. It supports multiple SCM tools
```

Comment:
Thanks! above `deprecated: true` would you also:

> Above the deprecated property add a comment noting that the chart is deprecated and linking to the deprecation policy.

Number 3 from https://github.com/helm/charts/blob/master/PROCESSES.md#deprecating-a-chart

Reply:
👍